### DISCUSSION TITLE
@vercel/node uses old typescript version

### DISCUSSION THREAD

**Discussion by ilrein** [vercel/community#1956] [Help]
### Summary

Currently uses: "typescript": "4.3.4"

Latest version is 5.0.3

Can we please update?

**Comment by TooTallNate** [comment]
You can add "typescript" as a dependency in your project's `package.json` file and `@vercel/node` will use that version instead.

**Reply by ilrein** [reply]
I tried this but it didn't work.

I was able to manually go into my global NPM packages and update @vercel/node to use the latest, now all my errors are gone.
